test/sqllogictest/types.slt

```diff
@@ -337,6 +337,23 @@ SELECT '1'::pg_catalog.oid
 ----
 1
 
+# 🔬🔬 regproc
+
+query T
+SELECT '1'::regproc
+----
+1
+
+query T
+SELECT '1'::pg_catalog.regproc
+----
+1
+
+query T
+SELECT 'array_in'::pg_catalog.regproc
+----
+1
```

Comment:
This is wrong, and was certainly just cruft I never cleaned up while writing a mock test. In postgres the oid is 750, but we don't support that function so it should be an error. The test should be changed to test a non-existent function (not `array_in`, because we'll need it for pgjdbc), and a function that does exist.